diagnostic.md

```diff
@@ -36,13 +40,17 @@ Now, we want to get a single donut from the server. Write the `curl` request
 you'd use to retrieve a single donut, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --request GET http://www.example.com/donuts/chocalate-frosted
 ```
```

Comment:
Right Idea but were were looking for an ID.  I think you get the point though
"curl --include --request GET http://www.example.com/donuts/42"